benchmark/inference/utils.py

```diff
@@ -20,9 +20,11 @@
 }
 
 
-def get_dataset(name, root, use_sparse_tensor=False, bf16=False):
+def get_dataset(name, root, use_sparse_tensor=False, bf16=False,
```

Comment:
Let's just use `remove_edge_index=True`. Not sure if we desperately need this additional argument.

Reply:
I think you meant remove_edge_index=False...  I've set it as default now.

Reply:
Yeah, sorry :)